src/main/java/it/gov/pagopa/payhub/ionotification/service/IOServiceImpl.java

```diff
@@ -12,11 +14,13 @@ public class IOServiceImpl implements IOService {
 
     private final IOServiceCreationService ioServiceCreationService;
     private final IONotificationService ioNotificationService;
+    private final IOManageService ioService;
 
     @Autowired
```

Comment:
Usage of `@Autowired` is deprecated and useless in this class. Constructor injection is good and sufficient.